tests/test_gparser.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import os
+import telemetry
+import time
+import datetime
+from unittest.mock import Mock
+
+TEST_JOB = 'http://server/jenkins/job/HW_tests/' \
```

Comment:
I think "server" needs to be a variable like: https://github.com/sdgtt/telemetry/blob/master/tests/test_elastic.py#L7
As it gets defined here: https://github.com/sdgtt/telemetry/blob/master/.github/workflows/test.yml#L33